src/transformState.js

```diff
@@ -5,7 +5,26 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const key of actions) {
```

Comment:
apples - apple
books - book
users - user
tables - table
actions - ?

```suggestion
  for (const key of actions) {
```